<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayersWindow</class>
 <widget class="QDockWidget" name="LayersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Слои</string>
  </property>
  <widget class="QWidget" name="mDockWidgetContents">
   <layout class="QVBoxLayout" name="mDockWidgetLayout">
    <item>
     <widget class="LayersTreeWidget" name="mLayersTreeWidget">
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>32</number>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>32</number>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">3</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mButtonLayout">
      <item>
       <spacer name="mButtonSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="mAddLayerGroupPushButton">
        <property name="toolTip">
         <string>Новая группа</string>
        </property>
        <property name="icon">
         <iconset resource="../res/resources.qrc">
          <normaloff>:/images/new_layer_group.png</normaloff>:/images/new_layer_group.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mAddLayerPushButton">
        <property name="toolTip">
         <string>Новый слой</string>
        </property>
        <property name="icon">
         <iconset resource="../res/resources.qrc">
          <normaloff>:/images/new_layer.png</normaloff>:/images/new_layer.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mDeletePushButton">
        <property name="toolTip">
         <string>Удалить</string>
        </property>
        <property name="icon">
         <iconset resource="../res/resources.qrc">
          <normaloff>:/images/delete_layer.png</normaloff>:/images/delete_layer.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LayersTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>layers_tree_widget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
